thirdparty/ditaa-mini/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>kroki-thirdparty</artifactId>
+    <groupId>io.kroki</groupId>
+    <version>0.4.1</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>io.kroki.thirdparty.ditaa</groupId>
+  <artifactId>ditaa-mini</artifactId>
+  <packaging>non-maven-jar</packaging>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>com.github.stephenc.nonmavenjar</groupId>
+        <artifactId>non-maven-jar-maven-plugin</artifactId>
+        <configuration>
+          <jarFile>${basedir}</jarFile>
```

Comment:
we don't have to explicitly define the jar name? it will take every *.jar files in `${basedir}`?

Reply:
It tries to match the closest to the artifact. If there is more than one jar file and the other jar files do not end in `-javadoc.jar` or `-sources.jar` then you need to specify which jar file

Reply:
This means that we can just replace the jar with the next version and not have to change the pom. Also means that we can leave the jar version number in the jar file name

Reply:
That's neat, thanks for the explanation 👍